### DISCUSSION TITLE
199

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#178] [General]
# 潮流周刊第199期 - 键盘鼠标

封面图拍摄于上周购买的罗技 MX Mechanical 茶轴键盘和 MX Master 3s 鼠标，均是 For Mac 白色版本，质感绝了很不错，白色也很简单，不过对于用习惯 Mac 自带键盘和触控板的我而言还需要适应一下，但是有点难适应这个过程。

https://weekly.tw93.fun/posts/199-%E9%94%AE%E7%9B%98%E9%BC%A0%E6%A0%87/

**Comment by lovingchina** [comment]
最后一条，作为十多年的 MacBook 用户，我的最终归宿是 NiZ 静电容键盘 + Apple 触摸板 + 支持 USB-C 反向充电的 4K 显示器（竖屏使用），然后在公司和家里各放置一套

**Reply by Konata9** [reply]
和老哥的配置差不多。不过我的键盘是 NuPhy，直接叠在 MacBook 上太爽了。

**Comment by hooray** [comment]
如果只是看日历，而用不到日程，可以试试TinyCal，很轻量

**Comment by ClishWang** [comment]
Pake 项目还更新新的app么？

**Reply by tw93** [reply]
有好用的就更新

**Comment by Hakunassan** [comment]
天空一道闪电，纠错别字人出现！
“上次后将获得一个链接传递使用。” -> “上传后将获得一个链接传递使用。”

**Reply by tw93** [reply]
哈哈哈 谢谢反馈

**Comment by hayden0319** [comment]
礼貌提问这个博客是哪个项目

**Reply by tw93** [reply]
https://github.com/tw93/weekly
